lib/build_cache.py

```diff
@@ -578,7 +593,7 @@ def tree_print(self):
       # See: https://git-scm.com/docs/git-log#_pretty_formats
       if (ch.verbose == 0):
          # ref names, subject (instruction)
-         fmt = "%C(auto)%d %Creset%<|(79,trunc)%s"
+         fmt = "%C(auto)%d %Creset%s"
```

Comment:
If limiting the summary output is a testing issue then perhaps `--tree` could interpret multiple verbosity flags.

no verbosity = truncate
-v = full summary, no truncating
-vv = full debug output (what -v does now with `--tree`)

We can then update tests to pass -v with --tree to achieve readable default behavior and a way to test properly.

Reply:
Yeah you convinced me to keep the truncation. I'll fix it.